earth_enterprise/src/fusion/gecombineterrain/gecombineterrain.cpp

```diff
@@ -339,8 +343,10 @@ int main(int argc, char **argv) {
 
     merger->Start();
     JOBSTATS_END(job_stats, MERGER_CREATED);
+    END_PROFILING(create_merger);
 
     // Feed this merge into a QuadsetGather operation
+    BEGIN_PROFILING(create_gatherer, "create", "gatherer");
```

Comment:
Maybe we should put our profiler within the JOBSTATS_ block to avoid accidentally including their overhead within our overhead